### ISSUE TITLE
New version of fan is not supported

### ISSUE THREAD

**Issue by cuplesoft** [ANYCUBIC-3D/Kobra_Neo#1]
New version of fan "HSC BCY4510D24E" is not fully supported. Fan not running from 0 to 94%
Probably issues with the PWM control.

**Comment by 1coderookie** [comment]
Nope, it's not a PWM control issue, it seems to be an issue of the fan itself not working properly with PWM - I did many tests which confirmed it. Not every fan works (properly) with PWM, and it seems like AC is using one of those kinds here now.   

*I reported that issue and my findings to AC already, but if you reach out to the support, they still claim that "it's normal" that a part cooling fan only runs in an on/off mode..*    

Only solution at this point: either get yourself a 4510 which is working properly with PWM, like the model which is being used previously ("Coolcox BF4510H24S" - you can see a picture of it [here](https://1coderookie.github.io/KobraGoNeoInsights/hardware/printhead/#neo_6) ), or get yourself a 5015 blower which is working properly with PWM and print a fan duct for it.

**Comment by NuclearPhoenixx** [comment]
~~Not sure what's up, but I don't have issues with my machine. I can set the fan speed any percentage and I can clearly hear a difference in speed. It's a new printer so I'm strongly assuming it's the same HSC BCY4510D24E fan.~~

Nevermind, I double checked and apparently still got a unit with the old fan.
